optuna/visualization/matplotlib/_matplotlib_imports.py

```diff
@@ -27,6 +30,9 @@
 
 __all__ = [
     "_imports",
+    "dates",
+    "ticker",
+    "patches",
```

Comment:
Since these modules are only used in the timeline plot. How about removing them and use `matplotlib.dates`, `matplotlib.patches` and `matplotlib.ticker` instead?

Reply:
Agreed. I revised the two files.